client/app/components/composites/SearchBar/SearchBar.css

```diff
@@ -45,137 +120,277 @@
    anything within it has focus, we have to create a special container
    for the focus styles to cover the whole form area. */
 .focusContainer::before {
+  display: none;
   content: '';
   position: absolute;
   top: 0;
   left: 0;
   right: 0;
   bottom: 0;
   border: 1px solid var(--SearchBar_borderColor);
-  border-radius: 10em; /* Big enough to be fully rounded */
   z-index: var(--SearchBar_focusContainerZIndex); /* Required to allow mouse event to the inputs/buttons. */
   transition: border-color 0.1s ease-in;
+
+  @media (--SearchBar_medium) {
+    display: block;
+    border-radius: 10em; /* Big enough to be fully rounded */
```

Comment:
@Gnito That results in this:

<img width="407" alt="screen shot 2016-06-20 at 12 52 20" src="https://cloud.githubusercontent.com/assets/53923/16190028/e25f380e-36e5-11e6-9490-81dc08543d84.png">

I think 50% would only work for square elements.